chad/README.md

```diff
@@ -0,0 +1,166 @@
+### Chad Miller
+### Answers are all inline, either after or below the question, beginning with //
+
+### Homework #1
+(Due Monday 7/20)
+
+You need to turn in this homework by cloning this repo, making a new branch with your solutions, pushing to github, and issuing a pull request.
+For details, follow the instructions [here](http://portlandcodeschool.github.io/jse/2015/01/07/command-line-and-git-slides/#/14).
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+---
+
+**1)** (_30% of total time_)
+Predict the output of each of the following expressions, and explain the rules which determine that answer.  (_Hint: The rules for operator `==` are complicated, but do your best!_)
+
+If the output depends on the value or type of variable _x_, identify the conditions (what _x_ is) when the expression will output true (or false, if that's simpler).  Assume the cases are independent, and _x_ is reset to an unknown value before each.
+
+Some of these are tricky!  Don't trust your first instinct.  
+
+
+**a)** `"1" == 1` //true / js will change types when using ==
+
+**b)** `"1" === 1` //false / js will NOT change types when using ===
+
+**c)** `x == 'x'` //false / unless x has been set to 'x'
+
+**d)** `x == (x+'')` //true / == ignores the added string
+
+**e)** `'' == ' '` //false / an empty string is not equal to a string with a space in it
+
+**f)** `x = true` //true / sets x to true
+
+**g)** `var x; x == 'undefined'` //false / x == undefined, not the string 'undefined'
+
+**h)** `'9'<'10'` //false / alphabetically 1 is "less" than 9
+
+**i)** `typeof x + 1 === "number"` //false / typeof is unary so operates on the first thing it sees to it's right. in this case typeof x is "number", but then a 1 gets concatenated on the end so it's not === to "number"
+
+**j)** `typeof x % 2 === "number"` //false / in this case typeof x starts out as "number" but then the % 2 adds uncertainty so js says NaN???
+
```

Comment:
The NaN in this case is caused by the % operator, which turns any value back as a number (which you see in the next question).